package.json

```diff
@@ -221,8 +221,12 @@
                 "title": "Node Health"
             },
             {
-                "command": "aks.aksKnownIssuesAvailabilityPerformanceDiagnostics",
-                "title": "Known Issues, Availability and Performance"
+                "command": "aks.aksCCPAvailabilityPerformanceDiagnostics",
+                "title": "Cluster and Control Plane Availability and Performance"
+            },
+            {
+                "command": "aks.aksStorageDiagnostics",
```

Comment:
Is this new command? or improvement of any existing one? Thanks.

I did read your docs, the only thing I wanted to check is that this is extra you think is good one to add as improvements or someone asked to do so? This looks harmless. (Worth to archive thought process)

Reply:
Yep, it's a new command that queries 5 new detectors related to Storage diagnostics. Added it as an improvement since it's a more recent diagnostic feature that's available on the portal but not currently in the extension. 👍
![AKS Diagnostics - Frame 4](https://github.com/user-attachments/assets/3f25c87b-fadb-4961-a2fd-05fe165000ab)

Reply:
Was the old command deprecated, and the new command includes the previous features and more?

Reply:
My above comment was referring to the new aksStorageDiagnostics command, I assumed that's the one Tats was referring to in the code snippet. In regards to the aksCCPAvailabilityPerformanceDiagnostics command, that's just a rename of the former aksKnownIssuesAvailabilityPerformanceDiagnostics command, which was using an outdated API endpoint. This was just a change in verbiage to reflect what's currently seen in the Azure portal & ARM API. 👍

Reply:
ahh ok, thanks for the clarification